docs/Tutorial_DAMP.ipynb

```diff
@@ -0,0 +1,408 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #21.        stumpy.config.STUMPY_EXCL_ZONE_DENOM = 1
</pre><p>According to paper, the exclusion zone is set to <code>m</code>. However, I decided to not hardcode it. In <code>STUMPY</code>,we may use the default setting and see how it affects the result.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/872/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>